TwittariaApp/src/main/java/tweets/Tweet.java

```diff
@@ -0,0 +1,120 @@
+package tweets;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Tweet {
+  private static final int DESCRIPTION_MAX_LENGTH = 280;
+  private static final int TAG_MAX_LENGTH = 20;
+  private final String id;
+  private String description;
```

Comment:
Why final ?
In case we make immutable object then everything should be final.
But in our case it will be easily to have mutable object.